app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def delete_resource(type, id, _data = {})
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.delete_in_provider_queue
```

Comment:
Is delete_in_provider_queue always available on Authentication resources ?  I only see those defined in manageiq/providers/ansible_tower/shared/automation_manager/*

Should we do a respond_to? first then do the new logic otherwise the current (super) path ?